ipatests/test_xmlrpc/test_cert_plugin.py

```diff
@@ -499,3 +506,34 @@ def test_revoke_with_reason_9(self):
 
     def test_revoke_with_reason_10(self):
         self.revoke_cert(10)
+
+
+@pytest.mark.tier1
+class test_cert_status(BaseCert):
+
+    @classmethod
+    def setup_class(cls):
+        super(test_cert_status, cls).setup_class()
+
+    """
+    Test the `cert-status` command.
+    """
+
+    def test_0001_cert_status(self):
+        """
+        Show status of certificate using request id
+        """
+        res = api.Command['cert_status'](request_id=1)['result']
```

Comment:
We will surely have `request_id=1` which is used for IPA CA certificate. That's why I used request id 1. We can change this if this is not the case.